<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">

<!--
# File: menuTools.ui
#
# Resource file for the Tools menu. It describes the Tools menu option buttons
# 
# Initial code isolated from previous file menu_casoCompleto.ui
#
#
#
# @dateCreated: 4-Aug-2018
# @authors: Arlem Aleida Castillo Avila, Felipe Orihuela-Espina
# @dateModified: 4-Aug-2018
#
# See also:
# 


#
# LOG:
# 4-Aug-2018: FOE: Added comments. Change text to English. Added CR for readability.
#
# 5-Aug-2018: FOE: The Manual Edit Segmentation Button now serves for
#   Start/Stop the operation
-->


 <class>Form</class>

 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>


  <widget class="QPushButton" name="paso0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0. Open image</string>
   </property>
  </widget>


  <widget class="QPushButton" name="paso1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1. Stitch / Mosaicing</string>
   </property>
  </widget>


  <widget class="QPushButton" name="paso2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2. Flattening</string>
   </property>
  </widget>


  <widget class="QPushButton" name="paso3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3. Segmentation</string>
   </property>
  </widget>


  <widget class="QToolButton" name="paso4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>4. Start manual edit segmentation</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>


 </widget>

 <resources/>
 <connections/>
</ui>
